recipes/ludoSim/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "ludoSim-jaib1" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e04071d847996209c3e0485dd814207619c3c3e3d404894fa531d03622f8c8d5
+  url: https://raw.githubusercontent.com/jaib1/ludoSim/master/LICENSE
+  sha256: 3972dc9744f6499f0f9b2dbf76696f2ae7ad8af9b23dde66d6af86c9dfb36986
```

Comment:
Because you are downloading two packages, you may not be in the root level of source code

Reply:
that second source url is just a direct source for the LICENSE (because the builder couldn't find it), if that's what you mean.

Here is the exact link for the tarball from the first source, when downloading and extracting the package is as I expect, but maybe I have done the organization wrong here? : https://pypi.io/packages/source/l/ludoSim-jaib1/ludoSim-jaib1-0.0.5.tar.gz

Reply:
right but conda-build might not br placing you in the correct directory. add an ls call to figure out where you at

Reply:
```
add an ls call to figure out where you at
```
sorry, could you tell me exactly where/how to do that? In the documentation, the only place I saw to run shell commands from the recipe was under `test: commands:`

Reply:
@scopatz Hi, just following up on this